terraform/gcp/provider.tf

```diff
@@ -1,12 +1,5 @@
 provider "google" {
-  credentials = file(var.credentials_path)
-  project     = var.project
-  region      = var.region
-}
-
-terraform {
-  backend "gcs" {
-    credentials = var.credentials_path
-    prefix      = "terragoat/${var.environment}"
-  }
```

Comment:
@JamesWoolfenden Why remove the BE?

Reply:
I dont think variables work in the init stage.

Reply:
It used to work...

Reply:
Theres no reference to those values, do you have env vars or a tfvars file locally? there are no vars set in this project or in the secrets?